src/TheThingsNetwork.h

```diff
@@ -51,7 +51,7 @@ class TheThingsNetwork
     void configureChannels(int sf, int fsb);
 
   public:
-    TheThingsNetwork(Stream& modemStream, Stream& debugStream, fp_ttn_t fp, int sf = TTN_DEFAULT_SF, int fsb = TTN_DEFAULT_FSB);    
+    TheThingsNetwork(Stream& modemStream, Stream& debugStream, fp_ttn_t fp = TTN_FP_EU868, int sf = TTN_DEFAULT_SF, int fsb = TTN_DEFAULT_FSB); 
```

Comment:
Don't initialize EU868 by default. The user has to specify it explicitly. We do not favor a region